src/nvim/fileio.c

```diff
@@ -5147,6 +5147,8 @@ static void vim_maketempdir(void)
   }
 }
 
+static char_u regexp_escape_chars[] = "[]{}";
```

Comment:
This is only used by  the `delete_recursive` function, it can be a local variable.